wezterm-gui/src/termwindow/mod.rs

```diff
@@ -1519,6 +1526,12 @@ impl TermWindow {
             } => {
                 // fall through
             }
+            MuxNotification::Alert {
+                alert: Alert::MouseCursorShapeChanged { .. },
+                ..
+            } => {
+                // fall through
+            }
```

Comment:
I copied what's done for `Alert::PaletteChanged` here, not sure if that's correct.